### DISCUSSION TITLE
[Feature] Share album with external immich users for collaboration

### DISCUSSION THREAD

**Discussion by smiba** [immich-app/immich#12565] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On a lot of online photo services (for example Google Photos) it's possible to make an album, and share this album with other users. 
These users can then add photos to the album, and see the album on their account as a separate shared album

Now this kinda already exists in the form of making a shared album, and allowing users to upload (and having a password on it).  However this is very different from my earlier example, as this album will not appear in "shared albums" within your local immich instance, there is no support for picking pictures from your instance and you need to manually enter a password (no authentication tokens are exchanged between instances). The album is entirely detached from your own Immich instance and there is no connection between these two.

It would be incredibly useful to have this, and it would make it much easier to convince friends to move to Immich as we usually make a big album with all photos whenever we're out and about together. 

Bonus points if we can invite people not by URL, but by simply inviting people using <username>@<immich instance>. (Similar to how you can interact on the FediVerse / Mastodon etc.)

I couldn't find anyone requesting this feature yet, but if it's a duplicate please let me know 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#728

**Reply by smiba** [reply]
Unfortunate to hear it's not being considered, it's in my opinion one of the only real downsides of Immich compared to any of the large competitors right now
